<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preWindow</class>
 <widget class="QMainWindow" name="preWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>预处理窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="binary_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>二值化显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok_Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>主摄像头</string>
    </property>
    <widget class="QSpinBox" name="dilate_spinBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>平滑系数：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="highTh_spinBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="erode_spinBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="dilate_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="erode_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="highTh_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>二值高阈值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>腐蚀系数：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="blur_spinBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>膨胀系数：</string>
     </property>
    </widget>
    <widget class="QSlider" name="lowTh_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="blur_horizontalSlider">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>二值低阈值：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="lowTh_spinBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>副摄像头</string>
    </property>
    <widget class="QSpinBox" name="dilate_spinBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>平滑系数：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="highTh_spinBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="erode_spinBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSlider" name="dilate_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="erode_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="highTh_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>二值高阈值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>腐蚀系数：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="blur_spinBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>膨胀系数：</string>
     </property>
    </widget>
    <widget class="QSlider" name="lowTh_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QSlider" name="blur_horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>二值低阈值：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="lowTh_spinBox_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>46</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>254</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="default_Button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复默认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
